products.html

```diff
@@ -0,0 +1,366 @@
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>FITBIT</title>
+
+  <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    integrity="sha256-eZrrJcwDc/3uDhsdt61sL2oOBY362qM3lon1gyExkL0=" crossorigin="anonymous" />
+
+
+  <link rel="stylesheet" href="https://suspicious-sammet-33f1b3.netlify.app/Components/components.css" />
+  <link rel="stylesheet" href="products.css" />
+  <link rel="stylesheet" href="common.css" />
+</head>
+
+<body>
+  <!--  ******************************** Navbar Start ********************************  -->
+  <nav class="navbar">
+    <div>
+      <img alt="hamburger-icon" src="images/hamburger.svg" class="hamburger-icon" id="hamburger-icon" />
+    </div>
+    <div class="logo-container">
+      <div class="logo">
+        <img class="logo-image" alt="logo-image" src="images/neatUI.svg" />
+        <span class="logo-text">FiT</span>
+        BiT
+      </div>
+    </div>
+    <div class="nav-items-container-end">
+      <div class="login-search-bar-container">
+        <button class="button button-primary">Login</button>
+        <div class="input-box">
+          <input type="text" class="input" value="" onchange="change(this)" required />
+          <label class="input-label">Search</label>
+        </div>
+      </div>
+
+      <div class="navbar-icons-container">
+        <a class="badge-container" href="#">
+          <i class="fa fa-heart-o" aria-hidden="true"></i>
+          <span class="number-badge number-badge-md">24+</span>
+        </a>
+        <a class="badge-container" href="#">
+          <i class="fa fa-shopping-cart" aria-hidden="true"></i>
+          <span class="number-badge number-badge-md">24+</span>
+        </a>
+      </div>
+    </div>
+  </nav>
+  <!--  ******************************** Navbar End ********************************  -->
+
+  <!--  ******************************** Products Section Start ********************************  -->
+
+  <section class="products-section">
+    <!--  ******************************** Sidebar Start ********************************  -->
+    <aside class="sidebar">
+      <div class="filters">
+        <p class="filter-type weight-semi-bold font-medium-large filter-heading">
+          FILTERS
+          <span class="weight-normal font-small">CLEAR</span>
+        </p>
+        <!--  Sort By Filter Start  -->
+        <div class="filter-type">
+          <p class="weight-semi-bold font-medium">SORT BY</p>
+          <div class="margin-vertical-lg"></div>
+          <ul class="list-style-none filter-list">
+            <li>
+              <input type="radio" name="sort-by" value="price-low-to-high" />
+              <label for="price-high-to-low">Price low to high</label>
```

Comment:
1) Price low to high has label for price-high-to-low
2) To Connect Input with label, you have to use id attribute not value
https://stackoverflow.com/a/6293626/13168852

Reply:
It is intensional

Reply:
> 1. Price low to high has label for price-high-to-low
> 2. To Connect Input with label, you have to use id attribute not value
>    https://stackoverflow.com/a/6293626/13168852

Ok understood i thought for connecting label with input we need name tag.